modules/end-user-guide/examples/snip_che-configuring-a-workspace-to-use-an-intellij-based-ide.adoc

```diff
@@ -0,0 +1,55 @@
+.Procedure
+
+. When using a JetBrains IDE requesting a registration, see xref:provisioning-jetbrains-activation-code-for-offline-use.adoc[].
+
+. Add the following component to the workspace devfile:
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+components:
+  - type: cheEditor
+    reference: "__<URL_to_meta.yaml>__" <1>
+----
++
+<1> __<URL_to_meta.yaml>__: HTTPS resource defining the editor metadata. It can be:
++
+* IntelliJ IDEA Community prebuilt image
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+components:
+  - type: cheEditor
+    reference: https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-idea/latest.meta.yaml
+----
++
+* PyCharm Community prebuilt image
++
+[source,yaml,subs="+quotes,macros,attributes"]
+----
+components:
+  - type: cheEditor
+    reference: https://raw.githubusercontent.com/che-incubator/jetbrains-editor-images/meta/che-pycharm/latest.meta.yaml
+----
++
+* Metadata for a custom editor image, see xref:building-intellij-editor-images.adoc[].
+
+. Remove the plugins or commands defined for the Theia editor from the workspace devfile.
+
+. Restart the workspace.
+
+. Read and accept the JetBrains Privacy Policy.
+
+. Make a choice in the IntelliJ data sharing preferences screen.
+
+. The IntelliJ IDE don't support checking out a repository using the devfile. Click the btn[New Project] to create a new project or the btn[Get from VCS] button to checkout a repository. 
```

Comment:
```
. The IntelliJ IDE does not support checking out a repository using the devfile. Click btn:[New Project] to create a new project or btn:[Get from VCS] to checkout a repository.
```

Reply:
Why an admonition block?

Reply:
Restiction valid only when not using the devworkspace enhgine. DevWorkspace projects are cloned by the operator before the editor is ran.

Reply:
IntelliJ IDE = IntelliJ Platform-based IDEs? or IntelliJ IDEA?

Reply:
> Why an admonition block?

This is a procedure step instructing the user to do something. It's confusing to read that something doesn't support something (because there is nothing the user can do about that), and then the second sentence instructs the user to do something (even though these two sentences appear to be partly related).

I guess my suggestion above can be rephrased without the admonition block like this: "Click btn:[New Project] to create a new project or btn:[Get from VCS] to checkout a repository because the IntelliJ IDE does not support checking out a repository using the devfile." But as the user do I really care to know that "because ..." - at which point, it becomes useful and optional information that lands in the admonition block.

Reply:
Added an admonition block at assembly level.